tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/tissue_mapper.py

```diff
@@ -123,60 +22,19 @@ def get_high_level_tissue(self, tissue_ontology_term_id: str) -> str:
             - If the input tissue is not found in the ontology, return the same as input.
                 - This could happen with something like "UBERON:0002048 (cell culture)"
         """
-        tissue_ontology_term_id = self.reformat_ontology_term_id(tissue_ontology_term_id, to_writable=False)
-
-        if tissue_ontology_term_id in self._cached_tissues:
-            # If we have looked this up already
-            return self._cached_tissues[tissue_ontology_term_id]
-
-        entity = self._get_entity_from_id(tissue_ontology_term_id)
-
-        if not entity:
-            # If not found as an ontology ID return itself
-            result = self.reformat_ontology_term_id(tissue_ontology_term_id, to_writable=True)
-            self._cached_tissues[tissue_ontology_term_id] = result
-            return result
-
-        # List ancestors for this entity, including itself. Ignore any ancestors that
-        # are not descendents of UBERON_0000061 (anatomical structure).
-        ancestors = [entity.name]
-        branch_ancestors: list[str] = []
-        for is_a in entity.is_a:
-            branch_ancestors = self._list_ancestors(is_a, branch_ancestors)
-
-        # Include this branch of ancestors is under anatomical structure
-        if self.ANATOMICAL_STRUCTURE_NAME in branch_ancestors:
```

Comment:
Is this functionality covered by COG?

Reply:
Not sure?

Reply:
this will not affect functionality. This is subsetting the ontology graph being built, but COG pre-builds the ontology graph and packages it in the library so it's not necessary, and is a superset of this graph. Census only matches terms to one of the terms in the curated list of tissue general terms anyway, so there isn't a concern around over-listing ancestors to users.